.travis.yml

```diff
@@ -7,8 +7,10 @@ cache:
 install:
   - npm ci
 script:
-  - npm run test
+  - npm run test -- --coverage
```

Comment:
Doesn't make sense to add `--coverage` to `npm run test` locally as well? So we can run tests and see what lines are not covered

Reply:
I wouldn't do that by default because it's quite a bit slower than running tests without coverage. You can still run `npm test -- --coverage` yourself to get coverage reporting.

Reply:
In terms of time, doesn't slow down that much. I got from 4s to 4.4s here. But indeed there is no need to add when we run `test`.

I'd add as a separate script `test:coverage` so we can recall ourselves to use during tests implementation if we want to keep the codebase well covered. 

And in case we want to run tests fast, use the normal `test` script, or even add a `test:watch` to watch file changes.

Reply:
Alright, that makes sense. I'll add `test:coverage`, watch is not really in this scope.